subgraph/schema.graphql

```diff
@@ -179,8 +191,9 @@ type Draw @entity(immutable: true) {
   dispute: Dispute!
   round: Round!
   juror: User!
-  voteID: BigInt!
+  voteIDNum: BigInt!
```

Comment:
Same here, I don't think the "Num" suffix is needed.

Reply:
So there is a collision in namespace when the graphql-codegen package generates types for graph queries.

There are Vote entities if an ID field [L31](https://github.com/kleros/kleros-v2/blob/42dfcd21c19e7256d9ed0d10ce8f4d298e0f6a5f/subgraph/schema.graphql#L31C10-L31C10). The generated types through errors, and more generally, its probably better practice to disambiguate the voteID here since it is an argument of an event, and not the id key of a vote entity.

I'm not sure a good alternative. . . . 

eventArgVoteID